bbb-common-message/deploy.sh

```diff
@@ -1 +1,5 @@
+scriptDir=$(dirname -- "$(readlink -f -- "$BASH_SOURCE")")
+rm -rf $scriptDir/../bbb-common-web/lib_managed/jars/org.bigbluebutton/bbb-common-message_2.12
```

Comment:
```suggestion
rm -rf $scriptDir/../bbb-common-web/lib_managed/jars/org.bigbluebutton/bbb-common-message_*
```

Could we keep it flexible about the bbb-common-message tag? Same on next line

Reply:
Yes!!